tools/cli/domainUtils.go

```diff
@@ -162,6 +162,14 @@ var (
 			Name:  FlagReason,
 			Usage: "Reason for the operation",
 		},
+		cli.StringFlag{
+			Name:  FlagFailoverTypeWithAlias,
+			Usage: "Domain failover type. Default value: force. Options: [force,grace]",
+		},
+		cli.StringFlag{
+			Name:  FlagFailoverTimeoutWithAlias,
+			Usage: "[Optional] Domain failover timeout in seconds. Default value: 60s",
+		},
```

Comment:
Probably `IntFlag` as the description says the value is in seconds? and default value "60" instead of "60s"?